sandbox.sh

```diff
@@ -4,18 +4,7 @@ set -e
 
```

Comment:
Most of the changes in this file seems to be shellcheck so would be nice to land #554 first